### DISCUSSION TITLE
pod count error

### DISCUSSION THREAD

**Discussion by u2bo** [k3s-io/k3s#3156] [Q&A]
**Environmental Info:**
K3s Version: v1.19.3+k3s2

Node(s) CPU architecture, OS, and Version: 
 3.10.0-957.el7.x86_64 #1 SMP Thu Nov 8 23:39:32 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

Cluster Configuration:
one node

**Describe the bug:**
some time update deployment image .and get 2 pod    looks like  2/1?

**Comment by brandond** [comment]
Can you provide an output showing what specifically you're seeing? I think you're talking about how the output shows that 1 of 2 containers in the pod are running, which is normal behavior for Kubernetes when things are starting up.
